test/test_polarion_fmftestcase_parser.py

```diff
@@ -113,7 +113,7 @@ def test_polarion_fmftestcase_parser_custom_parameters(testcase):
     :return:
     """
     assert testcase.project == 'ENTMQIC'
-    assert testcase.status == ''  # TODO Need to define how to handle it
+    assert testcase.status_id == 'approved'
```

Comment:
Is it correct to use "status_id"? I do not see it anywhere else.

Reply:
I got it from these jobs set up by Petr. And they seem to work. :)
https://mw-messaging-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/polarion-reporter/lastSuccessfulBuild/artifact/data/polarion.tc.xml